presidio-analyzer/Dockerfile

```diff
@@ -1,15 +1,18 @@
 FROM python:3.8-slim
 
 ARG NAME
-ARG SPACY_MODEL=en_core_web_lg
+ARG NLP_CONF_FILE=conf/default.yaml
```

Comment:
I would not change the dockerfile, but instead update the documetation with a sample of how to change it if a user wants to install NLP model in docker. WDYT?

Reply:
so you mean we wouldn't have an argument here? this could be useful if we want to build it with other versions for testing

Reply:
I would opt for just using the default, which is already configured in the code, so no need to add the config file.

Reply:
On second thought, let's leave it in the dockerfile, will make it easier for users to use the language config feature by just updating the config file and rebuilding the image. I'm resoving the comment

Reply:
These were my thoughts. Maybe it deserves more documentation to make sure users understand how to use it?